projects/1.21/assets/just-another-witchery-remake/witchery/lang/zh_cn.json

```
@@ -973,6 +1307,184 @@
   "tag.item.witchery.placeable_poppets": "可放置的娃娃",
   "tag.item.witchery.rowan_logs": "山梨木",
   "tag.item.witchery.to_spirit_world_transferable": "可带入精神世界",
+  "tarot.death.reversed.text": "腐朽吞噬着你。饥饿值会不断下降（每10秒0.5点消耗度）。夜晚到来时，死神本尊有15%概率前来索取你的灵魂。此牌在召唤后失效。",
+  "tarot.death.reversed.title": "死神 - 逆位",
+  "tarot.death.upright.text": "借由终结而转化。击杀可召唤的生物有5%概率令其复苏，成为24小时留存的空灵仆从。每次清晨都会清除所有负面效果。亡者复生。",
+  "tarot.death.upright.title": "死神 - 正位",
+  "tarot.judgement.reversed.text": "过往的罪恶永不消散。每次击杀都会对你造成2颗心的魔法伤害。遭谴者难以承受血之罪行的重量。",
+  "tarot.judgement.reversed.title": "审判 - 逆位",
+  "tarot.judgement.upright.text": "自余灰中复苏。受到致命伤害时，审判会有50%概率让你免受一死，留下1点生命值，并给予10秒的生命恢复 III、抗火、抗性提升 II。每次击杀恢复1颗心。神圣的慈悲有其限度——此牌在助你逃过死亡后失效。",
+  "tarot.judgement.upright.title": "审判 - 正位",
+  "tarot.justice.reversed.text": "不公残酷猖獗。天平已朝敌人倾斜，你受到的所有伤害增加30%。惩罚迅猛而无情。",
+  "tarot.justice.reversed.title": "正义 - 逆位",
+  "tarot.justice.upright.text": "完美的平衡得以维持。被你攻击过的生物对你造成伤害时，它们将受到2颗心的荆棘伤害。以眼还眼方显因果业报。",
+  "tarot.justice.upright.title": "正义 - 正位",
+  "tarot.strength.reversed.text": "虚弱充斥你的四肢。你的力量减弱了，攻击伤害减少1颗心。强者变得脆弱无能。",
+  "tarot.strength.reversed.title": "力量 - 逆位",
+  "tarot.strength.upright.text": "强大的威能流过全身。攻击伤害增加3颗心，每次击杀都会恢复2颗心的生命值。真正的力量遍及躯体和灵体。",
+  "tarot.strength.upright.title": "力量 - 正位",
+  "tarot.tarot_reading.page.1": "持有塔罗牌组右击以进行占卜。如此操作会抽出三张牌，揭示你在接下来三天内的命运。每张牌都有强大的效果——可以是正面的，也可以是负面的。水晶球会降低逆位牌出现的概率。按住左Alt可显示当前生效的牌。",
+  "tarot.tarot_reading.title.1": "塔罗牌占卜",
+  "tarot.temperance.reversed.text": "过度和失衡会阻碍运动。效果变得不稳定、难以预测（未实装）。适度与节制已完全消失。",
+  "tarot.temperance.reversed.title": "节制 - 逆位",
+  "tarot.temperance.upright.text": "完美的平衡能恢复和谐。生命值低于一半时，你会缓慢恢复生命值，每2秒恢复0.5颗心。水会给予水下呼吸。延长精酿的持续时间（未实装）。适度与节制能带来平静。",
+  "tarot.temperance.upright.title": "节制 - 正位",
+  "tarot.the_chariot.reversed.text": "你的道路将受阻。移动速度降低5%，如同在深泥地里行走一般。行走变成了疲乏的折磨。",
+  "tarot.the_chariot.reversed.title": "战车 - 逆位",
+  "tarot.the_chariot.upright.text": "如风般迅捷，如冲锋的骏马般不可阻挡。移动速度增加10%，赋予你追逐胜利的灵活。",
+  "tarot.the_chariot.upright.title": "战车 - 正位",
+  "tarot.the_devil.reversed.text": "脱离束缚。每30秒清除一次负面效果，但每次清除会丢失5级经验。自由的代价极为沉重。",
+  "tarot.the_devil.reversed.title": "恶魔 - 逆位",
+  "tarot.the_devil.upright.text": "黑暗力量须有牺牲。攻击伤害增加2颗心，但最大生命值降低1颗心。强大的力量以生命力为代价。",
+  "tarot.the_devil.upright.title": "恶魔 - 正位",
+  "tarot.the_emperor.reversed.text": "权威落为尘土。你的力量变得低微，每个清晨都将减少10级经验。控制力如沙尘般自指间流逝。",

```

```suggestion
  "tarot.the_emperor.reversed.text": "权威土崩瓦解。你的力量日渐衰微，每个黎明都将剥夺你10个经验等级。掌控力如流沙般从指缝间消逝。",
```
这里的10级经验是指0-10级的经验，还是当前等级直接-10

---

其实代码里是`player.giveExperiencePoints(-10)`
不是很懂作者为什么用level